unirec/utils/argument_parser.py

```diff
@@ -147,6 +147,11 @@ def parse_cmd_arguments():
     ## FM model
     parser.add_argument("--linear_mode", type=str, help='sparse linear mode in FM')
 
+    ## AdaRanker model
+    parser.add_argument("--train_type", type=str, default="Base", help='train type for AdaRanker')
```

Comment:
maybe we could use `choices` for the argument to indicate what train types we support.

Reply:
It is better to add those added arguments into `base.yaml` with default values.

Reply:
sounds great!